api/app/Notifications/ApplicationStatusChanged.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Enums\Language;
+use App\Enums\NotificationFamily;
+use App\Models\User;
+use App\Notifications\Messages\GcNotifyEmailMessage;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+
+class ApplicationStatusChanged extends Notification implements CanBeSentViaGcNotifyEmail
+{
+    use Queueable;
+
+    /**
+     * Create a new notification instance.
+     */
+    public function __construct(
+        public string $poolNameEn,
+        public string $poolNameFr,
+    ) {
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array<int, string>
+     */
+    public function via(User $notifiable): array
+    {
+        $notificationFamily = NotificationFamily::APPLICATION_UPDATE->name;
+        $enabledChannels = [];
+
+        if (! in_array($notificationFamily, $notifiable->ignored_email_notifications ?? [])) {
+            array_push($enabledChannels, GcNotifyEmailChannel::class);
+        }
+
+        if (! in_array($notificationFamily, $notifiable->ignored_in_app_notifications ?? [])) {
+            array_push($enabledChannels, 'database');
+        }
+
+        return $enabledChannels;
+    }
+
+    /**
+     * Get the array representation of the notification.
+     * https://laravel.com/docs/10.x/notifications#formatting-database-notifications
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(object $notifiable): array
+    {
+        return [
+            'poolName' => [
+                'en' => $this->poolNameEn,
+                'fr' => $this->poolNameFr,
+            ],
+        ];
+    }
+
+    /**
+     * Get the GC Notify representation of the notification.
+     */
+    public function toGcNotifyEmail(User $notifiable): GcNotifyEmailMessage
+    {
+        $locale = $this->locale ?? $notifiable->preferredLocale();
```

Comment:
Where does `$this->locale` come from?

Reply:
The notification can have its locale explicitly set when it is created.
https://laravel.com/docs/10.x/notifications#localizing-notifications